src/commands/lambda/flare.ts

```diff
@@ -29,6 +29,7 @@ const ENDPOINT_URL = 'https://datad0g.com/api/ui/support/serverless/flare'
 const FLARE_OUTPUT_DIRECTORY = '.datadog-ci'
 const LOGS_DIRECTORY = 'logs'
 const FUNCTION_CONFIG_FILE_NAME = 'function_config.json'
+const TAGS_FILE_NAME = 'tags.json'
```

Comment:
Is this a good name? Or should it be more descriptive like `resource_tags.json`?
Looks like AWS just calls them "Tags"

Reply:
Mmm, let's stick with what AWS calls them. These are indeed the resource tags, but the resource is Lambda, so, I think it makes sense to have them like this.